example/src/test/java/com/github/isopropylcyanide/example/app/JdbiUnitOfWorkModuleTest.java

```diff
@@ -1,56 +1,36 @@
 package com.github.isopropylcyanide.example.app;
 
+import com.github.isopropylcyanide.example.app.dao.CountingDao;
+import com.github.isopropylcyanide.example.app.resource.CountingResource;
 import com.github.isopropylcyanide.jdbiunitofwork.core.JdbiHandleManager;
-import com.google.common.collect.Sets;
 import com.google.inject.ConfigurationException;
 import com.google.inject.Guice;
 import com.google.inject.Injector;
+import jersey.repackaged.com.google.common.collect.Lists;
 import org.junit.Before;
 import org.junit.Test;
-import org.skife.jdbi.v2.Handle;
-import org.skife.jdbi.v2.sqlobject.SqlQuery;
-import org.skife.jdbi.v2.sqlobject.SqlUpdate;
 
-import java.util.Set;
+import java.util.List;
 
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThrows;
 import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
 
-@SuppressWarnings("unused")
 public class JdbiUnitOfWorkModuleTest {
 
     private Injector injector;
 
     @Before
     public void setUp() {
         JdbiHandleManager mockHandleManager = mock(JdbiHandleManager.class);
-        when(mockHandleManager.get()).thenReturn(mock(Handle.class));
-        Set<String> daoPackages = Sets.newHashSet("com.github.isopropylcyanide.example.app");
+        List<String> daoPackages = Lists.newArrayList("com.github.isopropylcyanide.example.app");
         JdbiUnitOfWorkModule module = new JdbiUnitOfWorkModule(mockHandleManager, daoPackages);
         injector = Guice.createInjector(module);
     }
 
     @Test
-    public void testModuleBindsTheProxiedDao() {
-        assertNotNull(injector.getInstance(DaoA.class));
```

Comment:
The fact that only `daoA and daoB` but not `daoC` is picked up, should be the responsibility of the library instead of the consumers. They should just be specifying their daos and that's it.